packages/http-client-csharp/emitter/src/lib/lib.ts

```diff
@@ -47,10 +47,9 @@ const diags: { [code: string]: DiagnosticDefinition<DiagnosticMessages> } = {
     },
   },
   "no-root-client": {
-    severity: "error",
+    severity: "warning",
```

Comment:
I don't think we should even warn for this case if there are models.

Reply:
Are you suggesting we remove this diagnostic? This no longer makes sense to me any more too

Reply:
I removed this diagnostic.
This also aligns with behavior in other language's generator.